docs/custom-model/openai.mdx

```diff
@@ -279,3 +279,7 @@ On execution, we get:
 | positive                               | I purchased this Kindle Fire HD 8 was purchased for use by 5 and 8 yer old grandchildren. They basically use it to play Amazon games that you download.                                                                                                                                                                                                                           |
 +----------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 ```
+
+<Note>
+Please note that when querying for predictions, the maximum number of rows that can be fetched at once is 20 (`LIMIT 20`) as it is a constraint of OpenAI.
```

Comment:
Oh, also forgot to mention: since yesterday this is no longer the case 😈 automated batching enables arbitrarily-sized queries to run, assuming you have enough tokens left in the account.